alchemist-graphql/build.gradle.kts

```diff
@@ -79,6 +79,9 @@ kotlin {
                 implementation(rootProject)
                 implementation(libs.bundles.graphql.server)
                 implementation(libs.bundles.ktor.server)
+//                implementation(incarnation("sapere"))
```

Comment:
```suggestion
```

Reply:
you may want to `testRuntime` it

Reply:
I can avoid to use it at all while there is no actual test upstream.

Reply:
ok